honeybee_radiance/sensorgrid.py

```diff
@@ -299,7 +295,7 @@ def __getitem__(self, index):
         return self.sensors[index]
 
     def __copy__(self):
-        new_obj = SensorGrid(self.identifier, self.sensors)
+        new_obj = SensorGrid(self.identifier, (sen.duplicate() for sen in self.sensors))
```

Comment:
You don't need to do this right now since the sensor object is immutable. Do you think that you will make the Senors mutable at some point? I imagine that it will be useful to scale the sensor grid to deal with different units systems or move it to different desk heights. Would adding this functionality break anything that you know of?

Reply:
Hmmm. Good question. Not that I can think of. It is part of a larger question of how we will deal with attaching results to a sensor or if we are even going to do that. If we copy a sensor does the results stays with the sensor or do you create a new sensor and wipe out the results? or do you create a new sensor and copy all the result. I don't have a good answer for this right now. What I do know is that the previous implementation made GH very slow and will not really be useful.